modules/infra/datapath/eth_output.c

```
@@ -18,72 +17,45 @@ enum {
-		} else if (iface_get_eth_addr(priv->iface, &src_mac) < 0) {
-			edge = NO_MAC;
-			goto next;
-		}
 
 		eth = gr_mbuf_prepend(mbuf, eth);
 		if (unlikely(eth == NULL)) {
 			edge = NO_HEADROOM;
 			goto next;
 		}
 		eth->dst_addr = priv->dst;
+		if (priv->iface->id != last_iface_id) {
```

Cppcheck (2.19.0)

[error] Uninitialized variable

(legacyUninitvar)